### PR TITLE
Arm64yk

### PR DESCRIPTION
**Pull request by ykarlsbrun**
Cosmetic changes for ajusty-cavium pull request
Changed test HashTest.cpp: for arm64 signed char should be mentioned explicitly
Test CpuId for arm64 is stub for now

### CODE REVIEW

**Comment by paulbiss** [comment]
cc @edwinsmith

**Comment by Gownta** [comment]
Thank you very much for your effort to make folly available on more platforms! Sorry for taking so long to comment on this patch. I've been going through the github pulls over the last few weeks and commenting on languishing requests.

We're working on getting folly continuously tested on different platforms. As soon as we do, we'll be able to properly evaluate and maintain this patch. In the interim, we'll keep this patch listed on github so that it stays on our radar

**Comment by paulbiss** [comment]
I'm assuming you meant to leave this open.

**Comment by Gownta** [comment]
Yes! Good catch, thank you. I must have pressed 'close and comment' instead of 'comment'

File: folly/test/CpuIdTest.cpp [hunk 1]
```diff
@@ -24,5 +24,9 @@ using namespace folly;
 TEST(CpuId, Simple) {
   // All CPUs should support MMX
   CpuId id;
+#ifdef __aarm64__
```
Comment (Orvid) [inline folly/test/CpuIdTest.cpp]: This doesn't quite seem right. Shouldn't it be the other way around?

File: folly/Portability.h [hunk 2]
```diff
@@ -273,4 +273,10 @@ inline size_t malloc_usable_size(void* ptr) {
 }
 #endif
 
+#if defined(__x86_64__)
```
Comment (Orvid) [inline folly/Portability.h]: This should be `#if defined(FOLLY_X64)`, which is what normally wraps the pause calls.

**Comment by sgolemon** [comment]
Rebased and applied the notes so far while pulling: D2152868

**Pull request closed without merging**